mobile/library/common/extensions/cert_validator/platform_bridge/platform_bridge_cert_validator.h

```diff
@@ -1,17 +1,21 @@
 #pragma once
 
-#include <thread>
+#include "envoy/thread/thread.h"
 
 #include "source/extensions/transport_sockets/tls/cert_validator/default_validator.h"
 
 #include "absl/container/flat_hash_map.h"
 #include "library/common/extensions/cert_validator/platform_bridge/c_types.h"
+#include "library/common/macros/macros.h"
+#include "library/common/thread/posix_thread.h"
```

Comment:
Somehow I couldn't find where this file is https://github.com/envoyproxy/envoy/tree/main/mobile/library/common/thread.

Reply:
That's a new file created in this PR.

Reply:
This is a stale comment.  I made the changes in `source/common/common/posix/thread_impl.cc` instead.